app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -146,8 +147,8 @@ class HistoryFragment : LibraryPageFragment<History>(), UserInteractionHandler,
             }
         }.toSet()
 
-    private fun invalidateOptionsMenu() {
-        activity?.invalidateOptionsMenu()
+    private fun invalidateMenu() {
+        activity?.invalidateMenu()
```

Comment:
I think we can revert this name change in order to keep the diff minimal - `invalidateOptionsMenu` seems fine to me